src/Enums/HTMLTags.hack

```diff
@@ -0,0 +1,179 @@
+namespace HTMLPurifier\Enums;
+
+enum HtmlTags: string {
```

Comment:
so there are many more tags in here than default `info`. Tried to do a quick comparison based on some other notes I had and looks like these are not included in the default `info` config:
```
"html", "base", "head", "link", "meta", "style", "title", "body", "figure", "data", "rb", "rp", "rt", "rtc", "ruby", "time", "area", "map", "video", "applet", "embed", "noembed", "object", "param", "canvas", "noscript", "script", "button", "datalist", "fieldset", "form", "input", "label", "legend", "meter", "optgroup", "option", "output", "select", "textarea", "details", "menuitem", "content", "element", "shadow", "slot", "template", "bgsound", "blink", "command", "frame", "frameset", "image", "isindex", "keygen", "listing", "marquee", "multicol", "nextid", "nobr", "noframes", "plaintext", "spacer", "xmp"
```

If you add some of these to a policy, what happens?

Reply:
Atm, if you add these to a policy you'll get the following error:
`Element 'X' is not supported at the moment. Please implement the element you would like to support, and add the element to the HTMLDefinition constructor.`

On the one hand, I think it makes sense to keep these enums with the idea that we _should_ support them, we just might not yet. On the other hand, I see how having things break is undesirable. What do you think adding an `HH_FIXME[1002]` to the unsupported flags, and then adding them in another ticket/PR?